{
 "bad": [
  "MCFReadError",
  "__dict_merge",
  "__parse_mcf_dict_recursive",
  "__to_dict",
  "collections.Mapping",
  "os.listdir",
  "os.path.dirname",
  "os.path.join",
  "os.path.realpath",
  "pygeometa.__init__.__version__",
  "pygeometa.__init__.cli"
 ],
 "deps": {
  "MCFReadError": [],
  "__dict_merge": [],
  "__parse_mcf_dict_recursive": [],
  "__to_dict": [],
  "click": [
   "click/conda-forge/noarch/click-7.0-py_0"
  ],
  "codecs": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "collections": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "configparser": [
   "configparser/conda-forge/noarch/configparser-5.0.0-py_0",
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "datetime": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "io": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "jinja2": [
   "jinja2/conda-forge/noarch/jinja2-2.10-py_1"
  ],
  "logging": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "os": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "pkg_resources": [
   "setuptools/conda-forge/linux-64/setuptools-41.2.0-py27_0"
  ],
  "pygeometa": [],
  "re": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "xml": [
   "python/conda-forge/linux-64/python-3.6.11-h4d41432_0_cpython"
  ],
  "yaml": [
   "google-cloud-bigquery-storage-core/conda-forge/noarch/google-cloud-bigquery-storage-core-2.0.0-pyh9f0ad1d_1",
   "pyyaml/conda-forge/linux-64/pyyaml-5.4.1-py36h70b1f00_0"
  ]
 }
}